extensions/integration-tests/src/onboarding/developers.ts

```diff
@@ -21,6 +21,7 @@ export const developers: string[] = [
 	'lucyzhang929',
 	'saiavishkarsreerama',
 	'smartguest',
+	'stutijain',
```

Comment:
This would go in a separate PR typically

Reply:
sure will do that

Reply:
done